apps/ledger-live-desktop/static/i18n/en/app.json

```diff
@@ -6601,7 +6601,8 @@
         "title": "Discover Ordinals",
         "description": "Do you know that you may own valuable rare sats and inscriptions?",
         "learnMore": "Learn more",
-        "viewCoinControl": "View coin control"
+        "protectTitle": "Protect your ordinals assets",
+        "protectDescription": "Exclude your Ordinals assets from your transactions"
```

Comment:
may be a `.` is missing at the end of the description

Reply:
Added ✅